receiver/windowseventlogreceiver/README.md

```diff
@@ -27,7 +27,7 @@ Tails and parses logs from windows event log API using the [opentelemetry-log-co
 | `resource`                          | {}           | A map of `key: value` pairs to add to the entry's resource.                                                                                                                                                                                    |
 | `operators`                         | []           | An array of [operators](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/README.md#what-operators-are-available). See below for more details                                                            |
 | `raw`                               | false        | If true, the windows events are not processed and sent as XML.                                                                                                                                                                                 |
-| `exclude_providers`                 | []           | One or more event log providers to exclude from processing.                                                                                                                                                                                        |
+| `exclude_providers`                 | []           | One or more event log providers to exclude from processing. Each event log will be processed in order to determine its provider name.                                                                                                          |
```

Comment:
I think this additional clarification is more appropriate on the `raw` field because that is where users are expecting to maximize performance. e.g. `If used in combination with `exclude_providers`, each event will be processed in order to determine its provider name.`